src/styles/_header.scss

```diff
@@ -17,20 +16,15 @@
 
 .nav {
   &__list {
-    display: flex;
-    margin: 0;
-    padding: 0;
+    display: grid;
+    grid-template-columns: repeat(8, auto);
+    column-gap: 20px;
```

Comment:
there should be 4 cards

Reply:
![image](https://user-images.githubusercontent.com/51528776/183146544-6d2580de-c381-4e96-b2b0-113c82b010ba.png)
why "4" - it is nav menu with 8 links

Reply:
![image](https://user-images.githubusercontent.com/51528776/183146717-9826515e-1368-436f-9f0c-622cb692c973.png)